cloudfront/wellcomelibrary.org/edge-lambda/src/catalogueApi.ts

```diff
@@ -56,3 +56,25 @@ export async function* apiQuery(
     }
   }
 }
+
+export async function findWorkWithIdentifierValue(
+  identifierValue: string,
+  identifierType?: string
+): Promise<Work | undefined> {
+  const resultList = apiQuery(identifierValue);
```

Comment:
If you're filtering on an `identifier` value, you might want to use the `identifiers` filter.

Reply:
This is copy/pasted, but a nice idea, let me have a go.

Reply:
I've left this as is as it's used elsewhere, but will look at it after I've released this.